sdk/communication/Azure.Communication.Common/CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
 # Release History
 
-## 2.0.0-beta.2 (Unreleased)
+## 2.0.0 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
```

Comment:
For breaking change we can mention the following:
### Breaking Changes
- Introduction of `MicrosoftTeamsAppIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with a rawID starting with `28:`.